utils/land-patterns.stanza

```diff
@@ -1458,7 +1458,8 @@ public defn make-n-pin-soic-landpattern (num-pins:Int,                 ; number
                                          terminal-length:Toleranced,   ; the length of the terminals
                                          terminal-width:Toleranced,    ; the width of the terminals
                                          density-level:DensityLevel,   ; the density level of the design
-                                         exposed-metal-heat-feature?:False|Shape):    ; an optional shape of a thermal pad
+                                         exposed-metal-heat-feature?:False|Shape,    ; an optional shape of a thermal pad
+                                         pin-numbering:String):
```

Comment:
For `pin-numbering` it's a little more idiomatic to use a `pcb-enum` to represent the possible values, which is easier for tooling and guarantees correctness 

```
public pcb-enum PinNumbering : 
  SoicNumbering
  RowMajorNumbering
```

and then in the check

```
switch(pin-numbering) : 
  SoicNumbering : 
  RowMajorNumbering : 
``

Reply:
fixed